source/Core/Src/settingsGUI.cpp

```diff
@@ -531,19 +531,20 @@ static void displayPDNegTimeout(void) {
 
 static void displayUSBPDMode(void) {
   /*
-   * PD Mode
-   * 0 = Safe mode, no PPS, no EPR
-   * 1 = Default mode, tolerant + PPS + EPR
-   * 2 = Strict mode + PPS + EPR
+   * Supported PD modes:
+   *  DEFAULT,    1 = PPS + EPR + more power request through increasing resistance by 0.5 Ohm to compensate power loss over cable/PCB/etc.
```

Comment:
So, here is the thing: in my humble opinion [as not quite native speaker], _"compensate"_ will be way much more clear in this context than _"offset"_.

Reply:
Absolutely fine by me

Reply:
Great! Then consider it done.